src/auto-evo/AutoEvoExploringTool.cs

```diff
@@ -798,14 +904,212 @@ private void PlayWithCurrentSettingPressed()
+            c => (c - totalSpeciesAverage) * (c - totalSpeciesAverage)));
+
+        var speciesStillAliveList = worldsList.Select(w => w.CurrentSpeciesCount).ToList();
+        var speciesStillAliveAverage = speciesStillAliveList.Average();
+        var speciesStillAliveStandardDeviation = Math.Sqrt(speciesStillAliveList.Average(
+            c => (c - speciesStillAliveAverage) * (c - speciesStillAliveAverage)));
+
+        var speciesCountPerPatchList = worldsList.Select(w => w.PatchSpeciesCountAverage).ToList();
+        var speciesCountPerPatchAverage = speciesCountPerPatchList.Average();
+        var speciesCountPerPatchStandardDeviation = Math.Sqrt(speciesCountPerPatchList.Average(c =>
+            (c - speciesCountPerPatchAverage) * (c - speciesCountPerPatchAverage)));
+
+        var populationPerPatchList = worldsList.Select(w => (double)w.TotalPopulation / w.PatchesCount).ToList();
+        var populationPerPatchAverage = populationPerPatchList.Average();
+        var populationPerPatchStandardDeviation = Math.Sqrt(populationPerPatchList.Average(p =>
+            (p - populationPerPatchAverage) * (p - populationPerPatchAverage)));
+
+        var microbeSpeciesHexSizeList = worldsList.Select(w => w.MicrobeSpeciesAverageHexSize).ToList();
+        var microbeSpeciesHexSizeAverage = microbeSpeciesHexSizeList.Average();
+        var microbeSpeciesHexSizeStandardDeviation = Math.Sqrt(microbeSpeciesHexSizeList.Average(s =>
+            (s - microbeSpeciesHexSizeAverage) * (s - microbeSpeciesHexSizeAverage)));
+
+        var bbcode = TranslationServer.Translate("ALL_WORLDS_STATISTICS").FormatSafe(worldGenerations,
+            totalSpeciesAverage.ToString("F2", CultureInfo.CurrentCulture),
+            totalSpeciesStandardDeviation.ToString("F2", CultureInfo.CurrentCulture),
+            speciesStillAliveAverage.ToString("F2", CultureInfo.CurrentCulture),
+            speciesStillAliveStandardDeviation.ToString("F2", CultureInfo.CurrentCulture),
+            speciesCountPerPatchAverage.ToString("F2", CultureInfo.CurrentCulture),
+            speciesCountPerPatchStandardDeviation.ToString("F2", CultureInfo.CurrentCulture),
+            populationPerPatchAverage.ToString("F2", CultureInfo.CurrentCulture),
+            populationPerPatchStandardDeviation.ToString("F2", CultureInfo.CurrentCulture),
+            microbeSpeciesHexSizeAverage.ToString("F2", CultureInfo.CurrentCulture),
+            microbeSpeciesHexSizeStandardDeviation.ToString("F2", CultureInfo.CurrentCulture));
+
+        foreach (var organelle in SimulationParameters.Instance.GetAllOrganelles())
+        {
+            var percentage = worldsList.Average(w => w.MicrobeSpeciesOrganelleStatistics[organelle].Percentage);
+            var average = worldsList.Average(w => w.MicrobeSpeciesOrganelleStatistics[organelle].Average);
+            bbcode += TranslationServer.Translate("MICROBE_ORGANELLE_STATISTICS").FormatSafe(
+                organelle.Name.Replace('\n', ' '),
```

Comment:
I've seen this `Name.Replace(...)` in quite a few places now, so I think adding a new get only property in the organelle class named something like `NameWithoutSpecialCharacters` would be better to keep the logic in one place in case it ever needs to be updated to handle more special characters.

Reply:
I really want to ask why toxin valcume has a line break in it. Other organelles whose names are even longer are written in one line.

Reply:
It's purely for English text in the organelle selection buttons to look nicer. It'd be great if Godot had a space line break mode, but it doesn't (or didn't back then) so the next best thing was to force the line break at a known location.

If there's any other solutions to the GUI (that would also apply to other languages automatically) that would be great to find.

Reply:
I opened a separate issue about doing that properly: https://github.com/Revolutionary-Games/Thrive/issues/3985